developers/addons/addon.md

```diff
@@ -92,6 +92,15 @@ Notes:
 - A `discovery-method` may contain multiple `match-property` entries, and in such a case **all** entries must match i.e. it a logical `AND` function is applied.
 - If you want to apply a logical `OR` function you can define a second separate `discovery-method` containing the respective `match-property` entry.
 - Different add-on discovery finders may need different `discovery-parameters`. Check the JavaDoc of the respective finder. See the `mdns` example below.
+- The `match-property` has different allowed values for its `name` element depending on the discovery `service-type`, as shown in the table below.
```

Comment:
In the line above, could you add ```org.openhab.core.config.discovery.addon.*```? Maybe we could also link the javadoc build, but I think this is not intended.... https://ci.openhab.org/view/Documentation/job/openHAB-JavaDoc/javadoc/index.html

Reply:
@holgerfriedrich I was suggesting to refer to the class JavaDoc -- like [this one](https://github.com/openhab/openhab-core/blob/369bd228e7a361b1c884b9af5129304291243f91/bundles/org.openhab.core.config.discovery.addon.ip/src/main/java/org/openhab/core/config/discovery/addon/ip/IpAddonFinder.java#L58)